### PR TITLE
[skip-ci] format PHP code same as in owncloud/core

### PR DESCRIPTION
**Pull request by individual-it**
## Description
introduce the same code-style for PHP as in owncloud/core

## Motivation and Context
as we add more ocis specific API tests that are written in PHP we should take care about the code style in PHP

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+$dirIterator = new DirectoryIterator(__DIR__ . '/' . $dirToParse);
+
+$excludeDirs = [
+	'lib/composer',
+	'build',
+	'apps/files_external/3rdparty',
+	'apps-external',
+	'data',
+	'3rdparty',
+];
```
Comment (phil-davis) [inline .php-cs-fixer.dist.php:13]: oCIS is different - these are dirs that exist in core and need to be excluded

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	if ($fileinfo->isDir() && !$fileinfo->isDot()) {
+		$excludeDirs[] = $dirToParse . '/' . $filename;
+	}
+}
+
+$finder = PhpCsFixer\Finder::create()
+	->exclude($excludeDirs)
+	->notPath('config/config.php')
+	->notPath('config/config.backup.php')
+	->notPath('tests/autoconfig*')
```
Comment (phil-davis) [inline .php-cs-fixer.dist.php:26]: these are core things

**Review by phil-davis — COMMENTED** [review]
Are we going to add this to drone CI?

**Comment by individual-it** [comment]
@phil-davis I've deleted more oC10 specific things, please review again.
I don't think I would run that in CI currently, or what do you think? @micbar ?

**Review by phil-davis — APPROVED** [review]
We can decide later if we want to run it in CI. At least we have `make test-php-style` and can easily fix the style locally whenever we want.

**Pull request merged**